openerp/addons/base/res/res_bank.py

```diff
@@ -95,8 +97,51 @@ def _default_value(self, cursor, user, field, context=None):
                 value = address.get(field, value)
         return value
 
+    # Courtesy of lmignon (acsone)
```

Comment:
remove the comment line #courtesy of
(we should not put that for every contribution, the code would be a mess)
(you may put it in the commit message